website/content/partials/known-issues/aws-auth-external-id.mdx

```diff
@@ -0,0 +1,14 @@
+### AWS Auth Role configuration requires an external_id
+
+#### Affected Versions
+
+- 1.17.0 - 1.17.3
+
+#### Issue
+
+The Vault AWS Auth backend requires an external_id to be set during role configuration, or it will return a validation error.
+
+#### Workaround
+
+Configure the backend with an external_id set (if external ids are not being used, supplying any string longer than two
+characters will work). Then, configure any desired roles. After this is done, the external id can be removed.
```

Comment:
```suggestion
To avoid the error during configuration:

1. Set the `external_id` parameter when configuring AWS authentication plugin
   with a valid ID or any string longer than two characters.
1. Configure any desired roles. 
1. If you used an arbitrary string, remove the external ID.
```